apps/roles.go

```diff
@@ -0,0 +1,124 @@
+package apps
```

Comment:
should we put this in an acl package?

Reply:
When we talked about this before you brought up potentially having prebuilt routes that consumers can use, which is the pattern I've taken here since I kind of liked it. I'm not attached to it though, but I think splitting them could potentially create many packages if we use the same pattern in the future